doc/en/user/source/data/webadmin/workspaces.rst

```diff
@@ -164,3 +166,11 @@ The Security tab allows to set data access rules at workspace level.
 To create/edit the workspace's data access rules, check/uncheck checkboxes according to the desired role. 
 The Grant access to any role checkbox grant each role for any access mode.
 
+A new parameter has been introduced to override the security feature that is restricting access
```

Comment:
Documentation is long-lasting, stating something is "new" won't age well. Just state the parameter is there, and what it's for.